### ISSUE TITLE
Audio not working on iOS

### ISSUE THREAD

**Issue by TheBiggestNumber** [networked-aframe/networked-aframe#142]
When enabling audio (or using the 'Positional Audio' example), avatars do not appear to each other on iOS, Safari or Chrome.

**Comment by victorsh** [comment]
I don't believe this is an issue with the component it self. I think this is specifically related to how iOS handles audio. To get around this issue I used Howler.js, it plays audio on iOS out of the box. I haven't dived too deep into it's functionality, but it seems to bring a lot of features to web base audio. https://github.com/goldfire/howler.js/

**Comment by vincentfretin** [comment]
iOS had a lots of various audio issues since I started playing with all this since iOS 13.
Positional audio on iOS and macOS Safari still had issues with crackling audio even until iOS 15.4 / Safari 15.4 I think. Hubs disabled positional audio (PannerNode) for Safari because of those issues. The issues may have been fixed in 15.4 or 15.5, this should be retested.
Non positional audio is working properly if you're creating the AudioContext yourself and call the resume method on it via the touchend/mouseup events.

Looking quickly at the howler.js code and their spatial audio, they are using PannerNode too, but they handle properly resuming and suspending the AudioContext. I think this is the main point here. networked-aframe doesn't provide currently this logic in the core.
